bot/bot.py

```diff
@@ -72,6 +89,20 @@ async def close(self) -> None:
         if self.redis_session:
             await self.redis_session.close()
 
+    async def load_extension(
+        self,
+        name: str,
+        *,
+        package: Optional[str] = None,
+        ignore_cache: Optional[bool] = False
```

Comment:
Why is the default False?

Reply:
Since we load the cog earlier and then remove them from the cache (when using the `.c load` command, it would never be loaded as the cog would be present in the cache, so this option was added to force load the cog without checking the cache.

Since we don't want to force reload the cog when the bot initially loads the cogs, it is set to `False`.

Reply:
But this means that when `start()` calls `load_extension()` it will always load all cogs?

Because you do `if ignore_cache and not await self.unloads_cache.get(name):`?

I am a bit confused

Reply:
Ah yeah, I had fixed that locally during my last testing... forgot to push it. Should be fixed now in 8193125.